components/ArtPieces/index.js

```diff
@@ -1,3 +1,22 @@
+import { StyledContainer } from "./ArtPieces.styled"
+import ArtPiecePreview from "../ArtPiecePreview";
+
 export default function ArtPieces({ pieces }) {
-  return <div>Art Pieces</div>;
+  return( 
+  <StyledContainer>
+    {pieces && pieces.length > 0 ? (pieces.map((piece) => (
+      <ArtPiecePreview 
+        key={piece.slug}
```

Comment:
you could destructure pieces above, so you wouldn't need to type `pieces.` on every line.

`const {slug, name, artist, imageSource} = piece;`